user/enterprise/tcie-3.x-setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,288 @@
+    
+
+### System Requirements
+
+The standard setup consists of the *Travis CI Enterprise
+Platform* which hosts Kubernetes cluster with each service in separate pod, and one or more
+*Worker hosts* which run the tests/jobs in isolated containers using LXC
+and Docker.
+
+Each dedicated host or hypervisor (VMWare, OpenStack, or EC2) should run **Ubuntu 18.04** and have at least **8 gigs of RAM and 4 CPUs**. If the same host is to run as Worker host, inimum requirement would be **16 gigs of RAM and 8 CPUs**.
+
+If you're running on EC2, we recommend the **c4.2xlarge** instance type for both **Platform** and **Worker**. We also recommend using an image that uses EBS for the root volume, as well as allocating 80 gigs of space to it.
+
+For [high availability (HA)](/user/enterprise/high-availability/) configurations, you will also need self-hosted:
+
+* [Redis](https://redis.io/),
+* [RabbitMQ](https://www.rabbitmq.com/)
+* [Postgres](https://www.postgresql.org/)
+
+## 1. Setting up Enterprise Platform 
+
+The Travis CI Enterprise Platform handles licensing, coordinates worker
+processes, and maintains the Enterprise user and admin dashboard. It must be
+installed as Kubernetes cluster or on at least one machine instance acting as microk8s host 
+or multiple instances managed by Kubernetes, separate from that of the Travis CI
+Enterprise worker. We recommend **compute optimized** instance running
+Ubuntu 18.04 LTS or later as the underlying operating system.
+
+> If you are migrating from Travis CI enteprise 2.x, please make sure to read and execute the [datbase migration instructions](/user/enterprise/tcie-3.x-migrating-db-from-2.x-to-3.x./) first.
+
+1. *On your infrastructure management platform*, create a Travis CI Platform Security Group.
+
+    If you're setting up your instance image or Kubernetes cluster for the first time, you most probably need to create some kind of
+    a Security Group. Create an entry for each port in the table below:
+
+    | Port | Service         | Description                                                                  |
+    |:-----|:----------------|:-----------------------------------------------------------------------------|
+    | 8800 | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
+    | 5672 | Custom TCP Rule | For RabbitMQ Non-SSL.                                                        |
+    | 4567 | Custom TCP Rule | For RabbitMQ SSL.                                                            |
+    | 3333 | HTTPS           | TCIE 3.x User administration Web application over HTTPS access.              |
+    | 443  | HTTPS           | Web application over HTTPS access.                                           |
+    | 80   | HTTP            | Web application access.                                                      |
+    | 22   | SSH             | SSH access.                                                                  |
```

Comment:
Please add another column explaining from where this traffic is coming from to ensure that users know how to securely configure their Firewalls / Security Groups

Reply:
Added more details to the description (inbound access) in subsequent commit